tests/CatalogTests/Helpers/UtilsTests.cs

```diff
@@ -109,5 +110,57 @@ private static MemoryStream GetPackageStream()
         {
             return TestHelper.GetStream("Newtonsoft.Json.9.0.2-beta1.nupkg");
         }
+
+        [Fact]
+        public void GetNupkgMetadataWithLicenseUrl_ReturnsLicenseUrl()
+        {
+            // Arrange
+            var stream = TestHelper.GetStream("Newtonsoft.Json.9.0.2-beta1.nupkg");
+            var metadata = Utils.GetNupkgMetadata(stream, packageHash: null);
+            var baseUrl = "http://example/";
+
+            // Actq
+            var graph = Utils.CreateNuspecGraph(metadata.Nuspec, baseUrl, normalizeXml: true);
+            var licenseFileTriples = graph.GetTriplesWithSubjectPredicate(
+                graph.CreateUriNode(new Uri(String.Concat(baseUrl, "newtonsoft.json.9.0.2-beta1.json"))),
```

Comment:
>new Uri(String.Concat(baseUrl, "newtonsoft.json.9.0.2-beta1.json") [](start = 36, length = 66)

Nit: make `new Uri(String.Concat(baseUrl, "newtonsoft.json.9.0.2-beta1.json")` into a local variable so it's not duplicated in 3 places

Reply:
Changed to local variable.